app/src/main/java/org/gdg/frisbee/android/api/PlusApi.java

```diff
@@ -10,4 +14,15 @@
 public interface PlusApi {
     @GET("people/{gplusId}?fields=image&key=" + BuildConfig.IP_SIMPLE_API_ACCESS_KEY)
     Call<ImageInfo> getImageInfo(@Path("gplusId") String gplusId);
+
+    @GET("people/{gplusId}?fields=image,aboutMe,tagline,urls,url,cover,displayName&key="
+        + BuildConfig.IP_SIMPLE_API_ACCESS_KEY)
+    @Nullable Person getPerson(@Path("gplusId") String gplusId);
+
+    @GET("people/{gplusId}/activities/public?fields="
+        + "nextPageToken,"
+        + "items(id,published,url,object/content,verb,"
+        + "object/attachments,annotation,object(plusoners,replies,resharers))"
+        + "&key=" + BuildConfig.IP_SIMPLE_API_ACCESS_KEY)
```

Comment:
I really like this interceptor. API keys and Auth related things usually done in interceptor anyways. That's what Square does